src/usr/local/www/diag_packet_capture.php

```diff
@@ -514,7 +534,7 @@ function fixup_host($value, $position) {
 
 		$cmd = "/usr/sbin/tcpdump -i {$selectedif} {$disablepromiscuous} {$searchcount} -s {$snaplen} -w {$fp}{$fn} " . escapeshellarg($matchstr);
 		// Debug
-		//echo $cmd;
+		// echo $cmd;
```

Comment:
While there, could you please remove this whole commented out block?

Reply:
removed